### DISCUSSION TITLE
Free positioned nodes no longer allow free positioning

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [freeplane/freeplane#982] [Help]
**Describe the bug**
When moving free positioned nodes, they only move along particular horizontal and vertical axes.

**To Reproduce**
Change a node to free positioned using right-click menu.
Attempt to move the node.

**Expected behavior**
In prior 1.11.1 alpha releases it worked fine. They could be positioned anywhere.

**Mind map files**
[flutterDefaultMain.zip](https://github.com/freeplane/freeplane/files/10578897/flutterDefaultMain.zip)
(The free positioned nodes are on the right side of the map.)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - FP: 1.11.1-pre11
 - Java Version: 17.0.5

**Comment by dpolivaev** [comment]
It is not a bug but a new feature.
You can have the old behavior if you hold a  shift key when you drag the nodes. 
Unfortunately, I do not know any way to communicate changes than making breaking changes.
Otherwise, you would never ask me this question.

**Comment by sgjohnson1981** [comment]
If you can't communicate changes wouldn't it be a better idea for a new feature to use the alternative key, lest people think it's a bug, as I did? You still need to communicate new features somehow, right?

**Comment by dpolivaev** [comment]
I don't know how to communicate new features: if they don't break anything people would not discover them.
